main.js

```diff
@@ -1,217 +1,206 @@
-  var winningSide = document.querySelector('#winning-side');
-  var gameResetButton = document.querySelector('#reset-button');
-  var errorMessage = document.querySelectorAll('.error');
-  var numberOfGuesses = 0;
-  var deleteWinnerCard;
-  var randomNumber = createNumber();
-  var minNumber = 0;
-  var maxNumber = 100;
-
-  function createNumber() {
-    return Math.round(Math.random() * 100);
-  }
-
-  function disableButtons() {
-    submitButton.setAttribute('disabled', true);
-    updateButton.setAttribute('disabled', true);
-    clearButton.setAttribute('disabled', true);
-  }
-
-  function resetGame() {
-    minRange.value = 0;
-    maxRange.value = 100;
-    setRange()
-    minRange.removeAttribute('disabled');
-    maxRange.removeAttribute('disabled');
+var challengerGuesses = document.querySelectorAll('.guess');
+var challengerInputField = document.querySelectorAll('.game-input');
+var challengerNames = document.querySelectorAll('.player');
+var clearButton = document.querySelector('#clear-button');
+var errorMessage = document.querySelectorAll('.error');
+var gameResetButton = document.querySelector('#reset-button');
+var guessHint = document.querySelectorAll('.guess-clue');
+var guessOneDisplay = document.querySelector('#guess-1-in-game');
+var guessTwoDisplay = document.querySelector('#guess-2-in-game');
+var inputForGame = document.querySelector('#input-for-game');
+var inputForRange = document.querySelector('#set-range-box');
+var maxRange = document.querySelector('#max-range-input');
+var minRange = document.querySelector('#min-range-input');
+var playerOneDisplay = document.querySelector('#challenger-1-in-game');
+var playerTwoDisplay = document.querySelector('#challenger-2-in-game');
+var rangeView = document.querySelectorAll('.range-view-js');
+var submitButton = document.querySelector('#submit-button');
+var updateButton = document.querySelector('#update-button');
+var winningSide = document.querySelector('#winning-side');
+var numberOfGuesses = 0;
+var deleteWinnerCard;
+var randomNumber = createNumber();
+var minNumber = 0;
+var maxNumber = 100;
+
+function createNumber() {
+  return Math.round(Math.random() * 100);
+}
+
+function disableButtons() {
+  submitButton.setAttribute('disabled', true);
+  updateButton.setAttribute('disabled', true);
+  clearButton.setAttribute('disabled', true);
+}
+
+function resetGame() {
+  minRange.value = 0;
+  maxRange.value = 100;
+  setRange()
+  minRange.removeAttribute('disabled');
+  maxRange.removeAttribute('disabled');
+  for (var i = 0; i < challengerNames.length; i++) {
+    challengerNames[i].removeAttribute('disabled');
+  };
+  clearFields();
+  gameResetButton.setAttribute('disabled', true);
+}
+
+function disableInput() {
+  if (challengerNames[0].value.length !== 0) {
     for (var i = 0; i < challengerNames.length; i++) {
-      challengerNames[i].removeAttribute('disabled');
+      challengerNames[i].setAttribute('disabled', true);
```

Comment:
small variation but makes sense to set it as true instead of removing it all together